src/adapters/index.ts

```diff
@@ -0,0 +1,67 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Context } from "../types/context";
+import { Database } from "../types/database";
+
+export function createAdapters(supabaseClient: SupabaseClient<Database>, context: Context) {
+  return {
+    supabase: {
+      repositories: {
```

Comment:
`repositories` is a bit odd name because these are basically issues

Reply:
The checks are being managed at the repository level. Notice the code later it checks the repository URL and then will loop down through all the issues if the conditions are fine based on the repository last check time, as I understand.

Reply:
When I started I did it repository based, but afterwards I did it issue based because it was more efficient and accurate to track by issue more than by repo. I will rename it.